spec/api/user_api/v1/documents_spec.rb

```diff
@@ -19,11 +19,27 @@
       }
     end
 
+    let!(:optional_params) do
+      {
+        metadata: {
+          country: Faker::Address.country
+        }
+      }
+    end
+
     it 'Checks if params are ok and returns success' do
       post '/api/v1/documents', headers: headers, params: params
       expect(response.status).to eq(201)
     end
 
+    it 'Creates document with optional params and returns success' do
+      post '/api/v1/documents', headers: headers, params: params.merge(optional_params)
+      expect(response.status).to eq(201)
+      document = Document.last
```

Comment:
we don't create variables in 'it' blocks
use `let()` instead
`let(:last_document) { Document.last }`
but I recommend to use
`let(:account_last_document) { account.documents.last }`